debian/flatpak.postinst

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+#DEBHELPER#
+
+if [ "$1" = configure ]; then
+    # Run a do-nothing command (it just lists configured remotes) for
+    # its side-effect of initializing the shared system-wide repository.
+    flatpak remote-list --system >/dev/null || :
+fi
+
+exit 0
+
+# vim:set sw=4 sts=4 et:
```

Comment:
We had intentionally removed this before - https://github.com/endlessm/flatpak/commit/b4429508650cafbdc49c4e93f33a889c41e62a13. However, I _think_ it's fine to bring it back. It will create /var/lib/flatpak in the ostree, but that gets masked at runtime by the OS /var, which the image builder (and tmpfiles.d) sets up with the proper symlinks.

Reply:
I'll verify that once a nightly image is built with this flatpak.